### PR TITLE
cmd/snap, snapctl: add column selectors to services

### PR DESCRIPTION
**Pull request by chipaca**
With this change, `snap{,ctl} services` grow a couple of options:

```
~$ snap services -h
Usage:
  snap services [services-OPTIONS] [<service>...]

The services command lists information about the services
specified, or about
the services in all currently installed snaps.

[services command options]
      --startup      Show only the Startup column
      --current      Show only the Current column

[services command arguments]
  <service>:         A service specification, which can be just
                     a snap name (for all services in the
                     snap), or <snap>.<app> for a single
                     service.

~$ snap services xkcd-webserver.xkcd-webserver
Service                        Startup  Current  Notes
xkcd-webserver.xkcd-webserver  enabled  active   -
~$ snap services --current xkcd-webserver.xkcd-webserver
active
~$ snap services --startup xkcd-webserver.xkcd-webserver
enabled
```

### CODE REVIEW

**Comment by chipaca** [comment]
Note the first commit is #6104.

File: cmd/snap/cmd_services_test.go [hunk 1]
```diff
@@ -174,52 +174,69 @@ func (s *appOpSuite) TestAppOps(c *check.C) {
...
-	c.Check(n, check.Equals, 1)
+`},
+		{
+			args: []string{"services", "--startup"},
+			out:  "enabled\nenabled\nenabled\n",
+		},
+		{
+			args: []string{"services", "--current"},
+			out:  "inactive\ninactive\nactive\n",
+		},
```
Comment (stolowski) [inline cmd/snap/cmd_services_test.go]: Could you please add a test case for the error on "--current --startup"?

**Comment by pedronis** [comment]
Two things:

* is this a direction we want to go?  (I fear the answer might be yes but we need to be careful)
* I was a bit surprised that this is really literally just the column values, no name, no headers (which the help doesn't clearly implies btw)

**Comment by chipaca** [comment]
as it's for use in scripts, there's no point in having the header just to force script writers to prune it back off.

Wish there were verbs though so we could have `--is-$verb` instead, and return failure if not (so the script can become `if snap service --is-$verb some.service; then ....`

**Comment by chipaca** [comment]
bah, there's no strong reason for `--current` and `--startup` not to return 1 if 'no'. In fact they could print nothing and just do the exit code thing.

**Comment by pedronis** [comment]
> bah, there's no strong reason for `--current` and `--startup` not to return 1 if 'no'. In fact they could print nothing and just do the exit code thing.

but you can pass many services to it, would they AND among them?

also it gets even further from the expectations around the base command

**Comment by chipaca** [comment]
> but you can pass many services to it, would they AND among them?

looking at `systemctl`'s `is-active` and `is-enabled`, it ORs them: `Returns an exit code 0 if at least one is active, or non-zero otherwise.` and `Returns an exit code of 0 if at least one is enabled, non-zero otherwise.` Also they have `--quiet` :-)

**Comment by chipaca** [comment]
Another alternative is to add `is-active` and `is-enabled`-alikes just to `snapctl`.

**Comment by pedronis** [comment]
marking blocked until we agree on some patterns here, as discussed with @chipaca

**Pull request closed without merging**